origin-growth/src/resources/email.js

```diff
@@ -81,8 +114,7 @@ async function sendInvites(referrer, recipients) {
     }
 
     // Send the invite code to the recipient.
-    const targetUrl = `${dappUrl}/#/welcome/${code}`
-    const { subject, text, html } = generateInviteEmail(referrerName, targetUrl)
+    const { subject, text, html } = generateEmail(referrerName, targetUrl)
```

Comment:
looks like this should be:
```
const { subject, text, html } = generateEmail('invite', referrerName, targetUrl)
```

Reply:
ah yeah. good catch ! thank you for being such a thorough reviewer of my bad code :)

Reply:
Well you catch a lot of bugs in my code... Saves a lot of pain with debugging down the line